### ISSUE TITLE
Cannot unpair 

### ISSUE THREAD

**Issue by Solusan** [grover/homebridge-dacp#17]
Hello,

I made some test connecting with homebridge-dacp plugin with AppleTV and now I'd need to delete some "remote app" and let only one or delete all, but simply I can't.

Does anybody knows how can I make it?

Thanks :)

**Comment by grover** [comment]
You can remove the pairing through the remote control settings on the Apple TV. You might need to remove all there and pair the one you’d like to keep again.

**Comment by CooperCGN** [comment]
@Solusan Are you using the beta of tvOS 11.3. on your Apple TV? I had the same problem with it and had to completely reset my ATV 4K to get rid of the pairing. It seems that the 11.3 beta is still really buggy, after a while homebridge-dacp stopped responding or responds took forever, on top the play/pause button on the remote repeatedly stopped working (although pressing the touch pad still play/pauses the content). This happened to me when pressing it rapidly while I was testing the dacp-plugin.

**Comment by grover** [comment]
I don’t have access to the iOS betas. If you’re able to get me more details that would help tremendously. A homebridge log of this would be perfect if rum with DEBUG=* and -D on the command line.

**Comment by CooperCGN** [comment]
I don't think it has something to do with your plugin. I checked the logs by myself and everything seems to be working fine. I rather assume that at some point the beta glitches out, why else would my Touch Remote stop responding to pressing the play/pause button (happened in regular iTunes movies and also on Netflix and Prime). Maybe the changed a lot of the code because of AirPlay 2? After a restart of the ATV everything is back to normal.

**Comment by CooperCGN** [comment]
Just did some more testing, it seems that at some point (still haven't found out the reason) the ATV stops communicating properly with homebridge. When pressing play/pause on my iPad in the home app it's processed by homebridge (see log) but the ATV doesn't respond to it anymore. Vice versa any change on the ATV isn't passed to homebridge anymore. After a reboot of the ATV everythings back to normal.
[log.txt](https://github.com/grover/homebridge-dacp/files/1684843/log.txt)

**Comment by Solusan** [comment]
Hi, @grover :)

I' really lost with homebridge-dacp. When I intalled and configured for the first time, was great. No problems. Suddenly I tried to make some test with another homebridge server I have and after that... nothing is working and I cannot unpair the "homebridge remote's" that homebridge made.

That's crazy.. now there's a lot of "Remote control" that's not working and no one can be deleted.

@CooperCGN I'm using the stable branch :)

Thanks all in advance.

**Comment by grover** [comment]
Have you tried to remove them on the Apple TV? I'm travelling and can't guide you there at the moment.

**Comment by Solusan** [comment]
Hi, yes I tried. Many times within success. 

In the other hand the plugin was working fine and now is not working anymore.

Now I’m out but I’ll try to give you the log make it with “Homebridge -D”

Thanks.  ^^

**Comment by grover** [comment]
I've been able to reproduce this issue today. Let's see what I can do to fix this.

**Comment by grover** [comment]
I wonder if @postlund can chime in here? Does unpairing work for pyatv?
